packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/message-printer.ts

```diff
@@ -6,26 +6,24 @@ import { printer } from 'amplify-prompts';
  * A factory function that returns a function that prints the "success message" after adding auth
  * @param print The amplify print object
  */
-export const getPostAddAuthMessagePrinter =
-  (print: any) =>
-  (resourceName: string, skipNextSteps: boolean = false) => {
-    print.success(`Successfully added auth resource ${resourceName} locally`);
+export const getPostAddAuthMessagePrinter = (resourceName: string, skipNextSteps: boolean = false) => {
```

Comment:
won't this update require a corresponding update anywhere `getPostAddAuthMessagePrinter` is used?

Reply:
this only gets called here